src/pages/About.js

```diff
@@ -186,7 +186,7 @@ class About extends React.Component {
                 <div className="mb-tiny bold">
                   Summary (SRS) data
                   <span className="italic ml-tiny regular">
-                    1960—2014 data available
+                    1960—2015 data available
```

Comment:
Should we just use the `MAX_YEAR` const here? That way the next time it is changed, all this stuff just sort of updates?